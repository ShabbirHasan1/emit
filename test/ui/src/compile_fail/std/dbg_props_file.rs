fn main() {
    emit::dbg!(file: "x");
}
