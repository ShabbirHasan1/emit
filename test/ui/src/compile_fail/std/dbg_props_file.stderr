error: `file` can't be used as a key in the `dbg` macro
 --> src/compile_fail/std/dbg_props_file.rs:2:16
  |
2 |     emit::dbg!(file: "x");
  |                ^^^^^^^^^
